frontend-react/src/hooks/UseLookupTable.test.ts

```diff
@@ -0,0 +1,63 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import { lookupTableServer } from "../__mocks__/LookupTableMockServer";
+import { LookupTables, ValueSet } from "../network/api/LookupTableApi";
+
+import useLookupTable, {
+    generateUseLookupTable,
+    GetLatestData,
+    GetLatestVersion,
+    getSenderAutomationData,
+} from "./UseLookupTable";
+
+jest.mock("@okta/okta-react", () => ({
+    useOktaAuth: () => {
+        const authState = {
+            isAuthenticated: true,
+        };
+        return { authState: authState };
+    },
+}));
+
+describe("test all hooks and methods", () => {
+    beforeAll(() => lookupTableServer.listen());
+    afterEach(() => lookupTableServer.resetHandlers());
+    afterAll(() => lookupTableServer.close());
+
+    test("test GetLatestVersion", async () => {
```

Comment:
can these test descriptions be a little more specific? maybe `GetLatestVersion returns expected version` etc?

Reply:
@doug-s-nava resolved.